.github/workflows/rust.yml

```diff
@@ -17,6 +17,17 @@ jobs:
       - name: Format
         run: cargo fmt -- --check
 
+  audit:
```

Comment:
Change to lint?

Reply:
I think we add it

Reply:
Well I thought they were similar enough they could go in the same job, but I wasn't sure what to name that job lol